src/sys/socket/mod.rs

```diff
@@ -241,6 +247,12 @@ impl SockProtocol {
     #[cfg(apple_targets)]
     #[allow(non_upper_case_globals)]
     pub const KextEvent: SockProtocol = SockProtocol::Icmp; // Matches libc::SYSPROTO_EVENT
+
+    /// Packet filter on IPv4 traffic
+    /// NOTE: placed here due to conflict (little endian arch) with SockProtocol::NetLinkISCI
```

Comment:
```suggestion
    // NOTE: placed here due to conflict (little endian arch) with SockProtocol::NetLinkISCI
```

This should be a code comment, not a doc comment. 

> conflict (little endian arch)

BTW, wouldn't this conflict always occur regardless of the byte order since they have the same value

Reply:
> BTW, wouldn't this conflict always occur regardless of the byte order since they have the same value

I was wrong, they only have the same value on Little-endian machines. With big-endian, EthIp is 2048